hack/xref-docker-options

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# Compare against docker options, see what (if anything) podman is missing.
+#
+
+# Hardcoded list of options we don't want to show. Some of these are
+# implemented in podman but hidden (so of course not in man pages);
+# some are not implemented and never will be (swarm).
+declare -a hidden=(
+    "docker --config"
+    "docker --context"
+    "docker --debug"
+    "docker --host"
+    "docker -D"
+    "docker -H"
+    "docker container create --gpus"
+    "docker create --gpus"
```

Comment:
I think `create --runtime` is another one. We have `--runtime` as a global flag in Podman, not a per-command flag.